data/campaigns/Liberty/scenarios/03_A_Strategy_Of_Hope.cfg

```diff
@@ -196,13 +196,14 @@
             speaker=Relana
             message= _ "I agree. But, Baldras, what will you tell him? We know something has been going badly wrong in Wesnoth of late, that more than ordinary corruption and incompetence is afoot. But when the effects fall on mere peasants, the wrong of it may not seem so obvious from a noble’s chair."
         [/message]
+        # TODO: play orcish war drums
```

Comment:
Nice idea!

Reply:
Would https://github.com/wesnoth/wesnoth/blob/ddfbb860d7d0d06bc52814900faafb65f55eedd8/data/campaigns/The_Hammer_of_Thursagan/scenarios/12_The_Underlevels.cfg#L440-L447 work? Asking both about the WML construct used and about the audio file used.

Reply:
That's fine for the WML. I have no idea about the audio file but I presume that it should be fine as long as it exists.

Reply:
I'll add that to my todo list.